src/azure-cli/azure/cli/command_modules/disconnectedoperations/__init__.py

```diff
@@ -0,0 +1,43 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#
+# Code generated by aaz-dev-tools
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.command_modules.disconnectedoperations._client_factory import cf_image
+from azure.cli.core import AzCommandsLoader
+
```

Comment:
Please input the _help.py here like this https://github.com/Azure/azure-cli/blob/3b706553b8d75c9882c6a4b01f89178b975f3cad/src/azure-cli/azure/cli/command_modules/monitor/__init__.py#L9

Reply:
Added